modules/core/src/main/java/org/apache/ignite/client/ClientCache.java

```diff
@@ -176,6 +297,24 @@
      */
     public boolean remove(K key) throws ClientException;
 
+    /**
+     * Removes the mapping for a key from this cache if it is present.
+     * <p>
+     * More formally, if this cache contains a mapping from key <tt>k</tt> to value <tt>v</tt> such that
+     * <code>(key==null ?  k==null : key.equals(k))</code>, that mapping is removed.
+     * (The cache can contain at most one such mapping.)
+     *
+     * <p>Returns <tt>true</tt> if this cache previously associated the key, or <tt>false</tt> if the cache
+     * contained no mapping for the key.
+     * <p>
+     * The cache will not contain a mapping for the specified key once the
```

Comment:
Minor: Looks like the line break is at the different position every time :)

Reply:
I just copied the javadoc from synchronous operations :)